### PR TITLE
preserving privacy flag when tagging keeps on the old website (i.e. bug ...

### PR DESCRIPTION
**Pull request by eishay**
...fix)
@andrewconner @ymatsuda 
I'm not a javascript expert so please review

### CODE REVIEW

**Comment by eishay** [comment]
tested locally and it seems to work ok

**Comment by andrewconner** [comment]
You'll need to change main.js in shoebox/public/, this is just a backup main.js for angular development. Should do the trick though.

**Comment by andrewconner** [comment]
I'll update the angular version of this to be explicit as well.

**Comment by eishay** [comment]
@andrewconner updated

File: kifi-backend/modules/shoebox/public/js/main.js [hunk 1]
```diff
@@ -3760,9 +3760,11 @@ $(function () {
 			collectionId: collId,
 			keeps: $keeps.map(function () {
 				var a = this.querySelector('.keep-title>a');
+				var isPrivate = this.querySelector('.keep-private.on') != null;
 				return {
-					title: a.title,
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/public/js/main.js:3764]: Weird, I wonder how tabs got in there. Yeah, spaces.

**Pull request merged**